javascript.js

```diff
@@ -0,0 +1,20 @@
+// Set variavles
+var user = []; //inline coment
+
+/**
```

Comment:
Typo inside comment in token 'First'. Possible corrections:first (47),list (0),hist (0)